ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -44,7 +32,13 @@ public class BottomSheetController: UIViewController {
         preconditionFailure("init(coder:) has not been implemented")
     }
 
-    /// A scroll view in `contentViewController`'s view hierarchy.
+    /// Top part of the sheet content that is visible in both collapsed and expanded state.
+    @objc public let headerContentView: UIView?
+
+    /// Sheet content below the header which is only visible when the sheet is expanded.
+    @objc public let expandedContentView: UIView
```

Comment:
curious if the design will evolve to show little bit of preview because it seems like bottomsheet currently have discoverability issue.

Reply:
Yeah discoverability is something we're constantly trying to figure out with designers. It might be very case by case, maybe I could add a little "nudge" animation which reveals the content when the app opens, we'll see.